### DISCUSSION TITLE
Welcome to selectize.js Discussions!

### DISCUSSION THREAD

**Discussion by risadams** [selectize/selectize.js#1621] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by incodenet** [comment]
Hi there. I found one issue:  there is no way to clear only one item from selected items.  Please review that bug. It is very important and useful feature.  Thank you.

**Reply by risadams** [reply]
See the removeOption()  method
